### ISSUE TITLE
SiteConfigLeftAndMain not pulling in JS for "Access" tab radio buttons

### ISSUE THREAD

**Issue by colintucker** [silverstripe/silverstripe-siteconfig#32]
Noticed today in 3.4.0 that if you load the CMS via /admin (i.e. Pages admin), and then click on Settings, the radio buttons function correctly on the Access tab because the CMS has already loaded `CMSMain.EditForm.js` (i.e. SiteConfig is loaded via Ajax):

![Access fields working](http://i.imgur.com/GVw8Sy7.png)

However, if you load /admin/settings directly via URL, or reload it in the browser, the radio buttons on the Access tab break because `SiteConfigLeftAndMain` is not requiring `CMSMain.EditForm.js`:

![Access fields broken](http://i.imgur.com/a9NfuJf.png)

You can see that the older `CMSSettingsController` class was pulling it in via `init()` in line 11:

```php
public function init() {
   parent::init();
   
   Requirements::javascript(CMS_DIR . '/javascript/CMSMain.EditForm.js');
}
```

I suspect this will need to be added to `SiteConfigLeftAndMain` to fix?

**Comment by tractorcow** [comment]
That seems like the right fix, would you like to please PR this for us? :D

**Comment by dhensby** [comment]
Please :)

**Comment by colintucker** [comment]
Oh you guys... you know I can't refuse those emoticons. :D

**Comment by tractorcow** [comment]
Fixed with #33
